ios/FluentUI.Demo/FluentUI.Demo/DemoHostingController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import FluentUI
+
+/// A specialized subclass of `FluentThemedHostingController` that can be used for SwiftUI demo screens.
+class DemoHostingController: FluentThemedHostingController {
+    init(rootView: AnyView, title: String, readmeText: String? = nil) {
+        super.init(rootView: rootView)
+        self.title = title
+    }
+
+    @objc required dynamic init?(coder aDecoder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    @MainActor required dynamic init(rootView: AnyView) {
+        super.init(rootView: rootView)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureAppearanceAndReadmePopovers()
+    }
+
+    // MARK: - Demo Appearance Popover
+
+    func configureAppearanceAndReadmePopovers() {
+        let settingsButton = UIBarButtonItem(image: UIImage(named: "ic_fluent_settings_24_regular"),
+                                             style: .plain,
+                                             target: self,
+                                             action: #selector(showAppearancePopover(_:)))
+        navigationItem.rightBarButtonItems = [settingsButton]
+        let readmeButton = UIBarButtonItem(image: UIImage(systemName: "info.circle.fill"),
+                                           style: .plain,
+                                           target: self,
+                                           action: #selector(showReadmePopover))
```

Comment:
should be `showReadmePopover(_:)`?

Reply:
Technically true! I'll fix it here and the two other classes that I copied this from.